README.md

```diff
@@ -1,5 +1,345 @@
+
+Note that Labs blog posts are a bit of an exception. Categorically they are
+content changes, but the content lives in the Git repo, so technically they are
+code changes, so they follow the process for code changes.
+
+This section will cover the process for each type of change.
+
+### Content changes (Storyblok)
+
+Content in Storyblok moves through several stages: drafting, reviewing, ready to
+publish, published, deployed. It's confusing, but "published" in our Storyblok
+configuration does not actually mean that the content has gone live on the
+corresponding public website. However, you should never publish any content in
+Storyblok if it is not ready for public presentation. In our configuration,
+publishing content in Storyblok is a signal to the rest of the team that says:
+this content is ready at any time to appear on the public website. If it's not
+ready, don't hit the publish button.
+
+The Storyblok process is covered in more detail in the GitHub repo's wiki. A
+simplified version is presented here.
+
+The process for creating, updating, or deleting content in Storyblok is the
+same. There are three major stages, each with its own steps.
+
+1. Editing
+   1. Make your changes in Storyblok.
+   2. Preview how your changes will look on the site using Storyblok's preview
+      iframe. You should see a yellow banner at the top of the page telling you
+      that you can see unpublished content.
+   3. Save your changes.
+2. Publishing
+   1. Subscribe to Slack channel `#vercel-notifications` if you are not already
+      subscribed.
+   2. Once your content changes are approved, click the publish button in
+      Storyblok. Remember, clicking "publish" means that you are certifying that
+      to the best of your knowledge, these changes should be ready to be
+      deployed at any time to the live website. When you click the publish
+      button, Storyblok will make a request to Vercel to start a preview
+      deployment.
+   3. Look for deployment notifications in Slack. Vercel will send a message to
+      the `#vercel-notifications` channel containing a link to the preview URL.
+   4. Visit the Vercel preview URL to review and double-check your changes.
+3. Going live
+   1. When you want your changes to go to the live production site, coordinate
+      with the dev team to open a merge pull request to the `main` branch on
+      GitHub.
+   2. When the Vercel GitHub app comments on the merge PR with a preview URL,
+      visit that preview URL to check your changes. (You should also be able to
+      get the Vercel preview URL from the Slack channel.)
+   3. If it all looks good, coordinate with the dev team to merge the PR to
+      `main`.
+   4. Once that PR is merged to main, wait for the production build to finish
+      deploying (check Slack channel for notifications), then check your changes
+      against the live site. You may need to clear your browser's cache before
+      you can see your changes.
```

Comment:
If you were counting, you would have noticed that this workflow asks a content editor to check their work three times after they've finished reviewing it in the Storyblok UI: once after hitting the "Publish" button, once after the develop-to-main PR is created, and then once after that PR is closed/merged/deployed to production. Is that too many?

Reply:
It's more than would be ideal, but I don't think it's wise to remove any of the three.

- The check after a Storyblok Publish lets the content editor check the `published` version without dev interaction
- The check of the develop->main PR is the only one that might be eliminated -- if this first one :arrow_up:  is rendered against `develop`, then this :arrow_left: *should* be the same. But, it's still a sensible check anyways.
- The check of the live site after a merge to `main` has to be done.